Makefile

```diff
@@ -77,6 +77,22 @@ run-ui:
 run-ui-arm:
 	COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile-arm" $(DOCKER_COMPOSE_CMD) up -d api-ui ui notificacions pending_publisher
 
+.PHONY: build
+build:
+	COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile" $(DOCKER_COMPOSE_CMD) build api pending_publisher
+
+.PHONY: build-arm
+build-arm:
+	COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile-arm" $(DOCKER_COMPOSE_CMD) build api pending_publisher
+
+.PHONY: build-ui
+build-ui:
+	COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_FILE="Dockerfile" $(DOCKER_COMPOSE_CMD) build api-ui ui notificacions pending_publisher
```

Comment:
Typo "notificacions". Commands will need to be [fixed](https://0xpolygon.slack.com/archives/C0409DDNV7V/p1682418991842709) too.

Reply:
What editor are you using? I would strongly recommend installing a spellchecker, e.g. [Code Spellchecker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) for VSCode.